tests/integration/api/v1/contents/[username]/[slug]/thumbnail/get.test.js

```diff
@@ -98,7 +97,7 @@ describe('GET /api/v1/contents/[username]/[slug]/thumbnail', () => {
         `${orchestrator.webserverUrl}/api/v1/contents/${defaultUser.username}/${rootContent.slug}/thumbnail`,
       );
 
-      const responseBody = await response.buffer();
+      const responseBody = await response.arrayBuffer();
```

Comment:
Não precisa necessariamente mudar isso, mas para manter o mesmo `responseBody`, deixar mais contida a mudança necessária pela remoção da `cross-fetch`, e evitar mudar o `expect`, eu usaria algo como:

```suggestion
      const responseBody = Buffer.from(await response.arrayBuffer());
```